.vscode/firefox_build_and_run.bat

```diff
@@ -0,0 +1,4 @@
+%~d0
```

Comment:
I will probably still need to remove those two, right?

Reply:
Yes, I think so.  Put them in .gitignore.

Reply:
or maybe just the tasks, but keeping the bat?

Reply:
managed to put it in package.json, so it should be ok now